spec/models/on/touch_spec.rb

```diff
@@ -20,6 +20,12 @@ module On
         )
         expect(record.versions.last.event).to eq("update")
       end
+
+      it "saves a object" do
+        record = described_class.create(name: "Alice")
+        record.touch
+        expect(record.versions.last.reify.name).to eq("Alice")
+      end
```

Comment:
On master branch, this spec passes on AR4.2 and AR5.0 but fails on AR5.1 and AR 5.2

Please see https://travis-ci.org/fukayatsu/paper_trail/builds/378643661 .